profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.webservice.inc

```diff
@@ -122,62 +138,72 @@ function FPFISPoetryIntegrationRequest($user, $password, $msg) {
     // Get main job in order to register the messages.
     $main_reference = 'MAIN_%_POETRY_%' . $reference;
     $language_job = $request->status['lgCode'];
-    $ids = _tmgmt_poetry_obtain_related_translation_jobs($language_job, $reference)->fetchAll();
+    $ids = _tmgmt_poetry_obtain_related_translation_jobs($language_job, $reference)
+      ->fetchAll();
 
     // Handling the case where we can't find the corresponding job.
     if (!$ids) {
-      watchdog("Poetry: Error message", "Callback can't find a job with remote reference !reference .",
-        array(
-          '!reference' => $reference,
-        ), WATCHDOG_ERROR);
+      watchdog(
+        "tmgmt_poetry",
+        "Callback can't find a job with remote reference !reference .",
```

Comment:
t() is applied on the watchdog messages when outputting them, e.g. in theme_dblog_message(), see https://api.drupal.org/api/drupal/modules!dblog!dblog.admin.inc/function/theme_dblog_message/7.x. @voidtek is right it should not be used as you suggested @tatarbj.